test-harness/server-lucee@5.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name":"cbvalidation-lucee@5",
+    "name":"lucee@5",
```

Comment:
can you leave the `cbvalidation` prefix on the serfver, it's important so commandbox doesn't override any named servers.

Reply:
Didn't know I had modified that.  Sorry.